core/fields/Field_Rich.php

```diff
@@ -161,6 +161,56 @@ public function display() {
 					document.addEventListener('click',function(e){
 						// click event handler for editor 
 
+						
+
+						if (e.target.nodeName==='A') {
+							
+							// only work on anchors inside editor (i.e. not toolbar or elswhere... :) )
+							let in_editor = e.target.closest('.editor');
+							if (in_editor) {
+								// remember dynamic editor/textarea
+								window.this_editor = e.target.closest('.control').querySelector('.editor');
+								window.this_textarea = e.target.closest('.control').querySelector('textarea');
+
+								//console.log('show anchor editor');
+								window.editor_anchor = e.target;
+
+								let href = e.target.getAttribute('href');
+								let text = e.target.innerText;
+								let classes = e.target.classList.value;
+								let target = e.target.getAttribute('target') ? e.target.getAttribute('target') : "";
+
+								/* console.log('URL',href);
+								console.log('TEXT',text);
+								console.log('CLASS',classes); */
+								
+								// iL=inputLabels, iI=inputIds, cV=currentValues=[], hL=helpLabels
+								let iL = ["URL", "Display Text", "Class", "Target"];
+								let iI = ["a_url", "a_text", "a_class","a_target"];
+								let cV = [href, text, classes, target];
+								let hL = ["", "", "","'_blank' for new window/tab, otherwise leave empty"];
+
+								function onCreate_a() {
+									// do nothing?
+								}
+
+								function onAdd_a() {
+									// update anchors
+									window.editor_anchor.innerText = document.getElementById('a_text').value;
+									window.editor_anchor.href = document.getElementById('a_url').value;
+									window.editor_anchor.classList.value = document.getElementById('a_class').value;
+									window.editor_anchor.target = document.getElementById('a_target').value;
+								}
+
+
+								function onCancel_a() {
+									// do nothing
+								}
+								window.live_editor = this_editor;
+								createModal(iL, iI, cV, hL, onCreate_a, onAdd_a, onCancel_a);
```

Comment:
re above two comments, could not null be put here/place anonymous inline functions? `(){}`